src/API.ts

```diff
@@ -1,48 +1,92 @@
-import { saveAs } from 'file-saver';
-import { createWriteStream } from 'streamsaver';
+// Remote
+import { transfer } from 'comlink';
+import { RemoteReadableStream } from 'remote-web-streams';
 
 // Types
-import { isFileList } from './typeGuards';
+
+// Local
 import {
-  compression,
+  Compression,
+  PenumbraAPI,
+  // PenumbraDecryptionWorkerAPI,
+  PenumbraDecryptionWorkerAPI,
   PenumbraFile,
-  PenumbraFiles,
-  RemoteResourceWithoutFile,
+  PenumbraTextOrURI,
+  RemoteResource,
 } from './types';
-
-// Local
 import { blobCache, isViewableText } from './utils';
-// import { getWorkers } from './workers';
+import { getWorkers, setWorkerLocation } from './workers';
 
-// const workers = getWorkers();
+const resolver = document.createElementNS(
+  'http://www.w3.org/1999/xhtml',
+  'a',
+) as HTMLAnchorElement;
 
-/** Retrieve and decrypt files */
-async function get(
-  ...resources: RemoteResourceWithoutFile[]
-): Promise<PenumbraFiles> {
-  // workers.decrypt.comlink;
-  console.error('penumbra.get() is unimplemented');
-  return ['', ''].map((s) => new File([s], s));
+/**
+ * Retrieve and decrypt files
+ *
+ * ```ts
+ * // Load a resource and get a ReadableStream
+ * await penumbra.get(resource);
+ *
+ * // Buffer all responses & read them as text
+ * await Promise.all((await penumbra.get(resources)).map((data: PenumbraFile) =>
+ *  new Response(data.stream).text()
+ * ));
+ *
+ * // Buffer a response & read as text
+ * await new Response((await penumbra.get(resource))[0].stream).text();
+ *
+ * // Example call with an included resource
+ * await penumbra.get({
+ *   url: 'https://s3-us-west-2.amazonaws.com/bencmbrook/NYT.txt.enc',
+ *   filePrefix: 'NYT',
+ *   mimetype: 'text/plain',
+ *   decryptionOptions: {
+ *     key: 'vScyqmJKqGl73mJkuwm/zPBQk0wct9eQ5wPE8laGcWM=',
+ *     iv: '6lNU+2vxJw6SFgse',
+ *     authTag: 'gadZhS1QozjEmfmHLblzbg==',
+ *   },
+ * });
+ * ```
+ */
+async function get(...resources: RemoteResource[]): Promise<PenumbraFile[]> {
+  if (arguments.length === 0) {
```

Comment:
is there a reason to use `arguments` here instead of `resources`?

Reply:
Nope, I definitely should have used `resources`. I remember that when I was first implementing this function I wasn't spreading `...resources`, which is probably why I used `arguments`.

This is now fixed!